README.md

```diff
@@ -7,6 +7,49 @@ height="42" />
 
 Next-generation WordPress eCommerce Experience for WordPress sites at Newfold Digital.
 
+
+## Module Responsibilities
+
+* Ecommerce module displays `Home or Dashboard` page for Brand plugin, with below options:
+  1. Live site preview for both, live and coming soon mode for WordPress site with view & edit site options
+  2. Next steps to setup your site/store and also link to view completed tasks
+  3. Links to your Hosting Brand Account
+  4. Link to 24/7 support to reachout to experts to build your site for you. 
+* Ecommerce module also displays `Store` option which gives snapshot of recent customer activity with details like total sales, net sales, recent orders, products sold, visitors, views and also a link to detailed `Analytics` page of Woocommerce plugin.
+* `Store` page also, allows customers to Purchase/Enable/Manage and learn more about YITH extended plugins like: 
+  1. YITH Booking and Appointment for WooCommerce
+  2. YITH WooCommerce AJAX Search
+  3. YITH WooCommerce Gift Cards
+  4. YITH WooCommerce Wishlist
+  5. YITH WooCommerce Customize My Account Page 
+  6. YITH WooCommerce Ajax Product Filter.
+* Under Store we have a `Products & Services` submenu option which provides following options: 
+  1. Easy access to Add new product/Import CSV of products into your store. 
+  2. Option to setup bookings & appointments or offer gift cards on purchases via YITH extended plugins.
+* Next submenu under Store is `Sales & Promotions`where customer can setup different promotional offers on products in their store. 
+  1. Here we display 13 different campaign setup options offered by `WonderCart` plugin under `All Campaigns` tab. 
+  2. There is also a `Settings` tab where users can customize look & feel of how sales campaigns are displayed on their Store page.
+* Next submenu under Store is `Payments` where we store owner can setup payments option accepted on purchases at Store. 
+  1. There 3 online payment options via payments gateways like PayPal, Stripe & RazorPay(only for India location). 
+  2. We also offer offline payments option via cheque, bank transfer or cash on delivery.
+* Final submenu under Store is `Store details` with following options:
+  1. Store location and complete address, email address of store owner and currency in which products/services are displayed on the store.
+  2. Under `shipping option` tab on the store details page you can connect to Shipping partner like, Shippo (based on your store location).
+  3. Under `tax settings` tab, store owner can choose to enable/disable sales tax. 
+
+
+## Critical Paths
+
+1. Ecommerce Module should display, `Home` page with site preview, links to Hosting brand a/c and links to pending & completed tasks to setup Ecommerce store.
```

Comment:
The site preview and links to hosting shouldn't be considered critical paths here as they aren't directly related to e-commerce functionality. Also, I don't think the completed tasks not showing is critical enough to consider a critical path. If a link didn't show or were wrong, we wouldn't do an emergency release to fix it.

Reply:
The coming soon not working is important functionality but I agree it's not something we'd do an out of cycle release for. And it shouldn't be in the ecommerce module anyway. There is planned work to move that elsewhere like coming soon module since it doesn't really have much to do with ecommerce per say.